### DISCUSSION TITLE
Status Checks are empty

### DISCUSSION THREAD

**Discussion by mike011** [codemagic-ci-cd/codemagic-docs#1660] [Q&A] [answered]
Hello, I'm in the process of setting Codemagic on a new repo, I have added the Webhook and run the build on main numerous times. But for some reason when trying to setup the branch protection rules for main, Github is not finding the workflow I created. I don't know why. What step am I missing? 

![image](https://github.com/codemagic-ci-cd/codemagic-docs/assets/705433/56606a2e-ed44-43dc-b53f-55e60b49d382)

**Comment by dtrdic** [comment]
Hey @mike011 this message indicates that the workflow has not been successfully triggered or completed within the last week.
Are you using build triggers to automatically trigger a build on codemagic? let's try in this direction, have you set up which events would you like to trigger your webhook?

**Reply by mike011** [reply]
I created a PR in the repo and the Webhook was successfully triggered:
![image](https://github.com/codemagic-ci-cd/codemagic-docs/assets/705433/6d35baa6-ae9f-4fc3-92b4-e794b8626ccb)
Also this is what my yaml looks like:
![image](https://github.com/codemagic-ci-cd/codemagic-docs/assets/705433/39ab1f5c-1295-4442-935f-5784faf01df0)
But it looks like something isn't setup correctly as I'm getting:
![image](https://github.com/codemagic-ci-cd/codemagic-docs/assets/705433/787c05a5-21da-4364-804d-d8e171cee82b)

**Reply by dtrdic** [reply]
i can see in your yaml that you are missing the push event. 
```
triggering:
   events:
      -push
      -pull_request
```
and make sure you select which events would you like to trigger with this webhook
![image](https://github.com/codemagic-ci-cd/codemagic-docs/assets/38918425/7b4b1229-2f3f-4190-aca7-9d790961d6bb)

**Reply by mike011** [reply]
By adding `push` is kicks the pull request build off twice and is still not reporting the status on the pull request. Webhook is setup for all events.
![image](https://github.com/codemagic-ci-cd/codemagic-docs/assets/705433/d75e7212-0f18-4254-a9a7-d246678c4d38)
Trying to add the status check is still empty.

**Comment by mike011** [comment, marked as answer]
I had to set the GitHub user who has access Codemagic to have write permissions. Originally they only had read permissions.

**Reply by dtrdic** [reply]
great, i'm glad that worked:) Cheers!

**Comment by raopsych** [comment]
Facing same issue.
